RandomizerMain.cs

```diff
@@ -160,9 +244,29 @@ void InventoryManager_AddItem(On.InventoryManager.orig_AddItem orig, InventoryMa
 
             LocationRO randoItemCheck;
 
-            if(itemId != EItems.TIME_SHARD) //killing the timeshard noise in the logs
+            if (itemId != EItems.TIME_SHARD) //killing the timeshard noise in the logs
```

Comment:
Unless we start doing something crazy like randomizing individual time shards or the big time shards, I explicitly don't want AP logic to touch the time shards. If that changes then the AP logic block would just be removed from this check.

Reply:
Yeah currently in advanced seeds timeshards are placed as items to give once all the other items are placed out, so those could be items that other games find for this game. Honestly though those single shards are not worth much effort to worry about. If someone for some reason mentions it we can at least say why they may not be receiving random shards from folks, its honestly trash checks anyway.

Reply:
Oh no that's completely supported by my code it's just done without needing to call this method since the ap items are received in a different check. To make them actually useful you receive APQuantity number of Time shards. It helps a lot with actually getting the shop upgrades in the run too